Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -128,8 +133,32 @@ final class PledgePaymentPlanOptionView: UIView {
         multiplier: 1.0
       )
     ])
-
+    
     self.termsOfUseButton.setContentHuggingPriority(.required, for: .horizontal)
+
+    self.ineligibleBadgeView.addSubview(self.ineligibleBadgeLabel)
+    self.ineligibleBadgeLabel.translatesAutoresizingMaskIntoConstraints = false
+    self.ineligibleBadgeLabel.setContentHuggingPriority(.required, for: .horizontal)
+    self.ineligibleBadgeView.setContentHuggingPriority(.required, for: .horizontal)
+
+    NSLayoutConstraint.activate([
+      self.ineligibleBadgeLabel.topAnchor.constraint(
+        equalTo: self.ineligibleBadgeView.topAnchor,
+        constant: 6
```

Comment:
We can probably throw these constants into the Constants enum you added in your [other PR](https://github.com/kickstarter/ios-oss/pull/2213#pullrequestreview-2496876671)